### PR TITLE
provides two more common errors (fix suggestions) for Webpacker::Manifest::MissingEntryError

### PR DESCRIPTION
**Pull request by jasonfb**
see https://stackoverflow.com/questions/52639256/webpackermanifestmissingentryerror

### CODE REVIEW

File: lib/webpacker/manifest.rb [hunk 1]
```diff
@@ -105,11 +105,14 @@ def manifest_type(pack_type)
...
-1. You want to set webpacker.yml value of compile to true for your environment
-   unless you are using the `webpack -w` or the webpack-dev-server.
-2. webpack has not yet re-run to reflect updates.
-3. You have misconfigured Webpacker's config/webpacker.yml file.
-4. Your webpack configuration is not creating a manifest.
+1. You forgot to install node packages (try either `npm install` or `yarn install`)
+2. Your app has code with a non-standard extension (like a `.jsx` file) but the extension is not in the `extensions` config in `config/webpacker.yml`
+3. You have set compile: false (see `config/webpacker.yml`) for this environment
+   (unless you are using the `webpack -w` or the webpack-dev-server, in which case maybe you aren't running the dev server in the background?)
+4. Webpacker has not yet re-run to reflect updates.
```
Comment (justin808) [inline lib/webpacker/manifest.rb:112]: `bin/webpack` not Webpacker?

**Comment by justin808** [comment]
@guillaumebriday We should merge after the comment is addressed.

File: lib/webpacker/manifest.rb (hunk 1 above)
Comment (jasonfb) [reply lib/webpacker/manifest.rb:112]: you're right it was `webpack` before my change on that line was idle and not intentional

**Comment by jasonfb** [comment]
this is mostly about #1 and #2; #3 (was previously #1) I just cleaned up the verbiage to match "Possible causes" as it was written out of style with the others.

File: lib/webpacker/manifest.rb [hunk 2]
```diff
@@ -105,11 +105,14 @@ def manifest_type(pack_type)
     def missing_file_from_manifest_error(bundle_name)
       <<-MSG
 Webpacker can't find #{bundle_name} in #{config.public_manifest_path}. Possible causes:
-1. You want to set webpacker.yml value of compile to true for your environment
-   unless you are using the `webpack -w` or the webpack-dev-server.
-2. webpack has not yet re-run to reflect updates.
-3. You have misconfigured Webpacker's config/webpacker.yml file.
-4. Your webpack configuration is not creating a manifest.
+1. You forgot to install node packages (try either `npm install` or `yarn install`)
```
Comment (justin808) [inline lib/webpacker/manifest.rb:108]: does `npm install` work with webpacker?

**Review by justin808 — COMMENTED** [review]
one question

LGTM otherwise

File: lib/webpacker/manifest.rb (hunk 2 above)
Comment (guillaumebriday) [reply lib/webpacker/manifest.rb:108]: It should but we never mentioned it anywhere so I suggest to remove it here too

**Review by guillaumebriday — CHANGES_REQUESTED** [review]
Could you remove the `npm install` please?

File: lib/webpacker/manifest.rb (hunk 2 above)
Comment (justin808) [reply lib/webpacker/manifest.rb:108]: @jasonfb this is good, but please remove the "npm" part.

@guillaumebriday I'll fix this if we don't hear from @jasonfb. This is a helpful fix.

**Comment by jasonfb** [comment]
@justin808 @guillaumebriday -- done. Can you tell me just out of curiosity why you are recommending against `npm` in favor of `yarn` (I use yarn myself cause I find it a little smoother). 

FYI... this was researched by (and thanks the folks who fell down this rabbit hole)...
https://stackoverflow.com/questions/54113179/rails-webpackermanifestmissingentryerror-in-homeindex/70326851?noredirect=1#comment124478710_70326851

**Comment by justin808** [comment]
Hi @jasonfb, a number of years ago, yarn worked better. It's hard to support both equally well. If you think these could be supported equally well, please submit a PR with your justifications.

**Comment by guillaumebriday** [comment]
Thanks @jasonfb and @justin808

**Pull request merged**